frontend/src/pages/ManageQuiz.jsx

```
@@ -62,7 +77,10 @@ const ManageQuiz = () => {
 								<button className="px-4 py-2 bg-blue-500 text-white rounded-lg hover:bg-blue-600 transition">
 									Update
 								</button>
-								<button className="px-4 py-2 bg-red-500 text-white rounded-lg hover:bg-red-600 transition">
+								<button
+									onClick={() => handleDelete(category.id)}
+									className="px-4 py-2 bg-red-500 text-white rounded-lg hover:bg-red-600 transition"
+								>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)